Readme.md

```diff
@@ -153,7 +153,7 @@ The response from `sendMessage` is either a Message record or an `error` (if sen
 
 ```ballerina
 //Send the message.
-gmail:Message|error sendMessageResponse = checkpanic gmailClient->sendMessage(userId,messageRequest);
+gmail:Message|error sendMessageResponse = checkpanic gmailClient->sendMessage(messageRequest, userId = userId);
```

Comment:
Please provide PR or commit if the `checkpanic` already has been addressed in the master branch.

Reply:
Code snippet is moved to Module.md from Readme.md . This is change is  here: https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/commit/5a1d71c7f012e587bed57a1e5d58905d006ef3fa